inst/test_msmslibraries/test_MsBackendMsmslibraries.R

```diff
@@ -0,0 +1,20 @@
+## assuming an object "be" exists with an initialises backend
```

Comment:
Please add here a little more documentation/notes, such as
- what is expected (variable name(s), what they should be
- how can this test be called from another package
- what is being tested by the tests within this file